setup.py

```diff
@@ -28,8 +28,8 @@ def get_version(rel_path):
     project_urls={
         "Bug Tracker": "https://github.com/icsm/pgmuvi/issues",
     },
-    author='Peter Scicluna',
-    author_email='peter.scicluna@eso.org',
+    author='Peter Scicluna', 'Kathryn Jones',
```

Comment:
Small change required, each entry should be one string with comma separation inside it

``` suggestion
    author='Peter Scicluna, Kathryn Jones',
```